Controllers/review.controller.js

```diff
@@ -1,13 +1,17 @@
-const express = require("express");
-const { fetchReview } = require("../Models/reviews.model");
-const app = express();
+const { fetchReview, fetchAllReviews } = require("../Models/reviews.model");
 
 exports.getReview = (request, response, next) => {
   fetchReview(request.params.review_id)
     .then((table) => {
-      response.status(200).send(table[0] );
+      response.status(200).send(table[0]);
     })
     .catch((err) => {
       next(err);
     });
 };
+
+exports.getAllReviews = (request, response, next) => {
+  fetchAllReviews().then((reviews) => {
+    response.status(200).send(reviews);
```

Comment:
reviews should be sent in an object with an appropriate key. An example of this can be found in your endpoints.json file.